uamqp/async_ops/client_async.py

```diff
@@ -528,7 +521,7 @@ async def _client_ready_async(self):
     async def _transfer_message_async(self, message, timeout):
         sent = await asyncio.shield(
             self.message_handler.send_async(message, self._on_message_sent, timeout=timeout),
-            loop=self.loop)
+            )
```

Comment:
do we need to replace this with `**self._internal_kwargs`?

Reply:
yep, good catch!